src/searching.md

```diff
@@ -382,7 +382,7 @@ you need to tell Anki not to treat them specially. This is called "escaping a ch
   `part:"after the colon"`.
 
 - `And`/`Or`\
-  To search for the words `or` and `and`, wrap them with double quotes. For example, `dog "and" cat` searches for "dog", "cat" and the word "and".
+  To search for the words "or" and "and", wrap them with double quotes. For example, `dog "and" cat` searches for "dog", "cat" and the word "and".
```

Comment:
```suggestion
  To search for the words `or` and `and`, wrap them with double quotes. For example, `dog "and" cat` searches for "dog", "cat" and the word "and".
```

Reply:
I understand your reasoning but double quotes are used as we're talking about the string of characters *to search* and not the search term itself even though we say "wrap them in double quotes".

Reply:
I'm afraid I find my suggested version easier to read than what you currently have.

<img width="716" alt="Screenshot 2024-09-07 at 4 13 50 PM" src="https://github.com/user-attachments/assets/70a62541-65b8-46af-99ac-d25c291736d6">

Reply:
"to search for these words"? it's in a list, shouldn't confuse people.

Reply:
Works for me.